apps/sms/test/marionette/lib/messages.js

```diff
@@ -51,43 +42,17 @@ var ConversationAccessor = require('./views/conversation/accessors');
 
         Composer: new ComposerAccessor(client),
```

Comment:
I was just surprised we access Composer directly, but that's all. If the longer term plan is to remove it eventually then it's ok. It's still very clean.